Automattic/jetpack
# Search: add API data for record meter

This PR connects the record meter chart up with real API data

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* On a site with an active Jetpack Search subscription, go to  `/wp-admin/admin.php?page=jetpack-search&features=record-meter`
* Check that the record meter chart data is accurate to the website (eg. number of posts, pages, plan tier limit) are correct, and if you add/remove posts/pages/custom post types that this is reflected in the chart. 

![image](https://user-images.githubusercontent.com/30754158/161553815-e4cf8e91-a2cd-4fdf-943e-7a57990bf8b2.png)
